akka-actor/src/main/resources/reference.conf

```diff
@@ -54,6 +54,11 @@ akka {
   # n: positive integer, number of dead letters that will be logged
   log-dead-letters = 10
 
+  # When log-dead-letters is on, this will renble the logging after configured duration.
+  # on: suspend the logging forever;
+  # n: positive interger, after which the logging will be re-enabled.
+  log-dead-suspend-duration = on
```

Comment:
Can't we make this a duration here as well?

Reply:
Keeping it on will keep the same behavior?

Reply:
Wouldn't `Duration.Infinite` here also keep the same behavior?

Reply:
so I change the "on" to "infinite"?